src/sys/node/node-worker-main.ts

```diff
@@ -4,16 +4,31 @@ import { EventEmitter } from 'events';
 
 import type * as d from '../../declarations';
 
+/**
+ * A class that holds a reference to a node worker sub-process within the main
+ * thread so that messages may be passed to it.
+ */
 export class NodeWorkerMain extends EventEmitter {
+  /**
+   * A handle for the OS process that is running our worker code
+   */
   childProcess: cp.ChildProcess;
   tasks = new Map<number, d.CompilerWorkerTask>();
   exitCode: number = null;
   processQueue = true;
-  sendQueue: d.MsgToWorker[] = [];
+  sendQueue: d.MsgToWorker<any>[] = [];
```

Comment:
For my own edification, why does the generic argument here have to be `any`?

Reply:
was just the easiest way to get it to pass - I might have to pass something like `"optimizeCss" | "prepareModule" | ..." instead which is just clunky